internal/auth/configuration_test.go

```diff
@@ -130,6 +130,57 @@ func TestEnvironmentOverridesConfiguration(t *testing.T) {
 				assertEq("foo-client-id", foo.ClientConfig.ID, t)
 			},
 		},
+		{
+			Name: "Test Sessions",
+			EnvOverrides: map[string]string{
+				"SESSION_LIFETIME": "1h",
+				"SESSION_KEY":      "abcdefg",
+			},
+			CheckFunc: func(c Configuration, t *testing.T) {
+				assertEq(1*time.Hour, c.SessionConfig.SessionLifetimeTTL, t)
+				assertEq("abcdefg", c.SessionConfig.Key, t)
+			},
+		},
+		{
+			Name: "Test Redis Sessions",
+			EnvOverrides: map[string]string{
+				"SESSION_REDIS_CONNECTION": "redis://master.example.com:6379",
+			},
+			CheckFunc: func(c Configuration, t *testing.T) {
+				redisConf := c.SessionConfig.RedisConfig
+				assertEq([]string{"redis://master.example.com:6379"}, redisConf.ConnectionURLs, t)
+			},
```

Comment:
I wonder if it's also worth adding in a call to RedisConfig.Validate() and/or RedisConfig.Enabled() here? It seems like it could be included fairly easily, and I think would help get the error cases within them tested.